cli/tsc_config.rs

```diff
@@ -248,13 +249,17 @@ impl TsConfig {
     }
   }
 
-  /// Return the current configuration as a `TranspileConfigOptions` structure.
-  pub fn as_transpile_config(
-    &self,
-  ) -> Result<TranspileConfigOptions, AnyError> {
-    let options: TranspileConfigOptions =
-      serde_json::from_value(self.0.clone())?;
-    Ok(options)
+  /// Return the current configuration as a `EmitOptions` structure.
+  pub fn as_emit_options(&self) -> Result<ast::EmitOptions, AnyError> {
+    let options: EmitConfigOptions = serde_json::from_value(self.0.clone())?;
+    Ok(ast::EmitOptions {
+      check_js: options.check_js,
+      emit_metadata: options.emit_decorator_metadata,
+      inline_source_map: true,
+      jsx_factory: options.jsx_factory,
+      jsx_fragment_factory: options.jsx_fragment_factory,
+      transform_jsx: options.jsx == "react",
+    })
```

Comment:
Nitpick: `ast::EmitOptions` should implement `From<Value>` instead of using this method